pkg/rpcserver/server_test.go

```diff
@@ -15,11 +15,12 @@
 package rpcserver
 
 import (
-	"github.com/matrixorigin/matrixone/pkg/logutil"
 	"os"
 	"testing"
 	"time"
 
+	"github.com/matrixorigin/matrixone/pkg/logutil"
+
```

Comment:
Delete this blank line please

Reply:
This is the format of goimports, which I think should be used consistently

Reply:
All right.